pkg/vulnsrc/azure/azure.go

```diff
@@ -0,0 +1,29 @@
+package azure
+
+import (
+	"path/filepath"
+
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/mariner"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+)
+
+var (
+	ErrNotSupported = xerrors.New("format not supported")
+)
+
+func NewVulnSrc() mariner.VulnSrc {
```

Comment:
You need to add this function to [this map](https://github.com/aquasecurity/trivy-db/blob/8e907467e9ab0047bab497f3a18b04b58745eeae/pkg/vulnsrc/vulnsrc.go#L37-L68)

Reply:
Thanks, done.